tripy/tests/spec_verification/op_doc_guide.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
```

Comment:
We can just name this file `README.md`. The path of its containing directory already implies the nature of its contents.